rsl/parse.py

```diff
@@ -53,6 +53,8 @@ class RSLParser(object):
               'ENDIF',
               'COMMENT',
               'WHERE',
+              'ORDERED_BY',
```

Comment:
consider using ORDER_BY and REVERSE_ORDER_BY

Reply:
done